packages/financial-templates-lib/src/logger/DiscordTransport.ts

```diff
@@ -28,18 +28,23 @@ export class DiscordTransport extends Transport {
       const body = {
         username: "UMA Infrastructure",
         avatar_url: "https://i.imgur.com/RCcxxEZ.png",
-        embeds: [{ title: `${info.at}: ${info.message}`, description: this.formatLinks(info.mrkdwn), color: 9696729 }],
+        embeds: [{ title: `${info.message}`, description: this.formatLinks(info.mrkdwn), color: 9696729 }],
       };
 
-      // Select webhook from escalation path. If no escalation, use default webhook url, if postOnNonEscalationPaths is
-      // true.Else, if postOnNonEscalationPaths is false and there is no escalation path, do not post. This enables the
-      // transport to be configured to only send messages that have a specific escalation path.
-      if (this.postOnNonEscalationPaths && this.escalationPathWebhookUrls[info.notificationPath]) {
+      // Either the transport is configured to send on undefined escalation paths (will use default path if path does
+      // not exist) or the transport has a defined escalation path for the given message's notification path.
+      let webHooks: string[] = [];
+      if (this.postOnNonEscalationPaths || this.escalationPathWebhookUrls[info.notificationPath]) {
+        // The webhook is preferentially set to the defined escalation path, or the default webhook.
         const webHook = this.escalationPathWebhookUrls[info.notificationPath] ?? this.defaultWebHookUrl;
 
-        // Send webhook request. This posts the message on discord.
-        await axios.post(webHook, body);
+        // If the webHook is an object it can contain multiple hooks within it. Else, it must be a single hook.
+        if (typeof webHook == "object") webHooks = webHook;
```

Comment:
nit: should you be testing for array rather than object? `Array.isArray(webhook)`

Reply:
also confused by this